experiments/modelfree/noisy_actions_and_obs.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+
+function wait_proc {
+    if [[ -f ~/ray_bootstrap_config.yaml ]]; then
+        # Running on a Ray cluster. We want to submit all the jobs in parallel.
+        sleep 5  # stagger jobs a bit
+    else
+        # Running locally. Each job will start a Ray cluster. Submit sequentially.
+        wait
+    fi
+}
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+OUT_ROOT=data/aws/score_agents
+TIMESTAMP=`date --iso-8601=seconds`
+
+MULTI_SCORE_CMD="python -m modelfree.multi.score with medium_accuracy "
+
+# Not sure what this is doing
+if [[ $# -eq 0 ]]; then
+    echo "usage: $0 <logdir> [logdir ...]"
+    exit 1
+fi
+
+# Make a directory for each of the experiments we'll be running, to store results in
+for dir in noisy_adversary_actions noisy_victim_actions noisy_victim_obs; do
+    mkdir -p ${OUT_ROOT}/${dir}/${TIMESTAMP}
+done
+# Rerun highest_win_rate and store the results in ADVERSARY_PATHS, which we
+# export as an environment variable which multi score will use
+ADVERSARY_PATHS=${OUT_ROOT}/normal/2019-05-05T18:12:24+00:00/best_adversaries.json
+
+export ADVERSARY_PATHS=${ADVERSARY_PATHS}
```

Comment:
You can do this in one line: `export ADVERSARY_PATHS=${OUT_ROOT}/normal/2019-05-05T18:12:24+00:00/best_adversaries.json`